src/bin/common/parse_command.ml

```diff
@@ -936,8 +975,32 @@ let parse_output_opt =
     Arg.(value & opt (some format_conv) None & info ["o"; "output"] ~docv ~doc)
   in
 
+  let model_type =
+    let doc =
+      Format.sprintf
+        "Control the output model type of the solver, $(docv) must be %s."
+        (Arg.doc_alts [ "value"; "constraint" ])
```

Comment:
The documentation is wrong, It is `constraints` and not `constraint`. Perhaps adding more information about these options? I didn't find the meaning of `constraints` in the doc directory also. Actually I would like to minimize some values in a generated models but I didn't find how to do that.

Reply:
Constraints & optimizing models are not the same. The `constraints` mode outputs the set of expressions alt-ergo managed to prove (cf Fun_sat.t, field `gamma`, extracted from `extract_prop_model`).
This is not the optimizer.